u8timeseries/backtesting/backtesting.py

```diff
@@ -91,9 +91,8 @@ def backtest_forecasting(series: TimeSeries,
         train = series.drop_after(pred_time)  # build the training series
         model.fit(train)
         pred = model.predict(fcast_horizon_n)
-        values.append(pred.values()[-1])  # store the N-th point
+        values.append(pred.univariate_values()[-1])  # store the N-th point
```

Comment:
we will eventually need to extend model forecasts (and backtesting) to multivariate forecasts.

Reply:
I will add this to my todo list for step 3